src/widgets/health-monitoring/cluster/HealthMonitoringClusterTile.tsx

```diff
@@ -46,12 +41,37 @@ export const ClusterStatusTile = ({
     (sum: number, item: ResourceData) => (item.running ? item.cpu * item.maxCpu + sum : sum),
     0
   );
+  const uptime = data.nodes.reduce(
+    (sum: number, item: ResourceData) => (item.running ? item.uptime + sum : sum),
+    0
+  );
 
   const cpuPercent = (usedCpu / maxCpu) * 100;
   const memPercent = (usedMem / maxMem) * 100;
 
+  const formatUptime = (uptime: number) => {
```

Comment:
You should be able to use dayjs for this?
https://day.js.org/

Reply:
I try to add, but it dont work, so i have leave it so